htdocs/emailcollector/class/emailcollector.class.php

```diff
@@ -2658,8 +2658,6 @@ public function doCollectOneCollector($mode = 0)
 									} else {
 										dol_syslog("One and only one existing third party has been found");
 
-										$thirdpartystatic->fetch($result);
```

Comment:
i dont undertsand the fix. If we reach this line of code, it means $result contains the ID of thirdparty found by findNearest function.
So we can now load the thirdparty.